dotnet-desktop-guide/winforms/controls/snippets/how-to-make-thread-safe-calls/cs/FormBackgroundWorker.cs

```diff
@@ -8,37 +8,36 @@
 using System.Threading.Tasks;
 using System.Windows.Forms;
 
-namespace project
+namespace project;
+
+public partial class FormBackgroundWorker : Form
 {
-    public partial class FormBackgroundWorker : Form
+    public FormBackgroundWorker()
     {
-        public FormBackgroundWorker()
-        {
-            InitializeComponent();
-        }
+        InitializeComponent();
+    }
 
-        //<Background>
-        private void button1_Click(object sender, EventArgs e)
-        {
-            if (!backgroundWorker1.IsBusy)
-                backgroundWorker1.RunWorkerAsync();
-        }
+    //<Background>
+    private void button1_Click(object sender, EventArgs e)
+    {
+        if (!backgroundWorker1.IsBusy)
+            backgroundWorker1.RunWorkerAsync();
```

Comment:
Should we add a comment that although this methods ends in `Async`, it does _not_ return a Task, thus cannot be awaited?